opm/autodiff/MultisegmentWell.hpp

```diff
@@ -163,6 +163,18 @@ namespace Opm
             }
         }
 
+        Scalar volumetricSurfaceRateForConnection(int cellIdx, int phaseIdx) const {
```

Comment:
If this function in `StandardWell` and `MultisegmentWell` are the same, you can implement this function in `WellInterface`. Please do put the implementation in `WellInterface_impl.hpp`. The well models are getting more and more functions, it is important to keep the head file as neat as possible.